addons/binding/org.openhab.binding.lametrictime/about.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
+        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"/>
+    <title>About</title>
+</head>
+<body lang="EN-US">
+<h2>About This Content</h2>
+
+<p>April 30, 2017</p>
+<h3>License</h3>
+
+<p>The openHAB community makes available all content in this plug-in (&quot;Content&quot;). Unless otherwise
+    indicated below, the Content is provided to you under the terms and conditions of the
+    Eclipse Public License Version 1.0 (&quot;EPL&quot;). A copy of the EPL is available
+    at <a href="http://www.eclipse.org/legal/epl-v10.html">http://www.eclipse.org/legal/epl-v10.html</a>.
+    For purposes of the EPL, &quot;Program&quot; will mean the Content.</p>
+
+<p>If you did not receive this Content directly from the openHAB community, the Content is
+    being redistributed by another party (&quot;Redistributor&quot;) and different terms and conditions may
+    apply to your use of any object code in the Content. Check the Redistributor's license that was
+    provided with the Content. If no such license exists, contact the Redistributor. Unless otherwise
+    indicated below, the terms and conditions of the EPL still apply to any source code in the Content
+    and such source code may be obtained at <a href="http://www.openhab.org/">openhab.org</a>.</p>
+
+
+<h3>Third Party Content</h3>
+<p>The Content includes items that have been sourced from third parties as set out below. If you
+    did not receive this Content directly from the openHAB community, the following is provided
+    for informational purposes only, and you should look to the Redistributor's license for
+    terms and conditions of use.</p>
+<p>
+    <em><strong>liblametrictime-java</strong></em>
+    <br/>
+    <pre>Apache 2.0 License</pre>
+    </br>
+    Source: https://github.com/syphr42/liblametrictime-java
```

Comment:
On this topic specifically, I feel quite strongly that the device API should remain separate. While there may not be any public users on GitHub, I am using this API in other projects. But more importantly, I very much believe in software reusability and would argue against this sort of bundling. Ideas like the Unix philosophy of doing one thing and doing it well and the importance of low coupling and high cohesion in software units are important to me. I also believe strongly in the Apache license and would like as much of my code as possible to be under that license. 

Since the API source is Apache 2.0 licensed, you are free to copy the code into OH if you choose to, but my vote is against it.

Reply:
Ok, understood and fine for me.
My only wish being that you do your best to keep the code base clean wrt intellectual property, i.e. that you make sure to have potential future contributors sign [the DCO or an CLA](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/) as it is best practice in open source projects.

Reply:
That's fair. I added the [DCO](https://github.com/syphr42/liblametrictime-java/blob/master/CONTRIBUTING.md) and installed a [DCO check](https://github.com/probot/dco).